azure_functions_worker/dispatcher.py

```diff
@@ -85,10 +85,10 @@ def load_bindings(self):
 
     @classmethod
     async def connect(cls, host, port, worker_id, request_id,
-                      connect_timeout, max_msg_len=None):
+                      connect_timeout):
         loop = asyncio._get_running_loop()
         disp = cls(loop, host, port, worker_id, request_id,
-                   connect_timeout, max_msg_len)
+                   connect_timeout)
```

Comment:
As you are changing the "max_msg_len", in a subsequent PR, let's start adding type suggestions for the parameters. I'll add a new issue for it.

```
def __init__(self, loop, host, port, worker_id, request_id,
             grpc_connect_timeout, grpc_max_msg_len: int = -1):
```

Reply:
Felt this bug in prod.  Was tough to troibleshoot.  How can we tell if our azure app has the latest hotfix bits?

Reply:
Hi @TroyWitthoeft, we finished rollout the hotfix last week. You can check if your function host version is upgraded to 2.0.13998 or 3.0.13978. These two versions should have this HotFix.

Reply:
Got it.  We're all set now.  We were able to Kudu/SSH into the box and examine the dispatcher.py to confirm if we had the latest bits.  A delete and redeploy got us the latest bits.  But, I am curious.  We dug around in the portal and couldn't find the functions host version listed anywhere.  Any tips?  Is it on a particular part of the blade? Available over REST?  Printed to logs on startup, etc?  Appreciate it.